### ISSUE TITLE
Split into separate repos and configure SingularityHub per repo for CI builds?

### ISSUE THREAD

**Issue by willfurnass** [rses-singularity/all-images#1]
See [here](https://github.com/rses-singularity/fsl-debian-stretch-singularity) for an example of where I've done this.  @twinkarma any thoughts on this?

**Comment by twinkarma** [comment]
Yes it's probably a good idea. I'll sort this out once I have some free time.

**Comment by willfurnass** [comment]
Happy to help with this.  NB hosting images publicly on singularity-hub.org only legitimate if we have the rights to redistribute the assets in those images.  May be problematic for e.g. cuDNN.
